src/utils/getUnhandledProps.js

```diff
@@ -9,7 +9,7 @@ const getUnhandledProps = instance => {
   return _.omit(instance.props, (val, key) => {
     const inPropTypes = _.has(instance.constructor.propTypes, key);
     const inDefaultProps = _.has(instance.constructor.defaultProps, key);
-    return inPropTypes || inDefaultProps;
+    return inPropTypes && !inDefaultProps;
```

Comment:
Important change here.  This util is used to get props for spreading.  We want all props that were not handled in the class definition.  In the case that you define a default only (no propType and no usage in the class), it should be considered "unhandled".

This allows setting default props and still having them spread.  Input does this with `type`.